filtertags/translation_en.yml

```diff
@@ -65,8 +65,8 @@ subject:
       NAME: Technology
     music:
       NAME: Music
-    norwegian:
-      NAME: Norwegian
+    first_language:
+      NAME: English
     english:
       NAME: English
```

Comment:
Denne burde vel fjernes for engelsk?

Reply:
Altså ta bort: 

```
english:
      NAME: English
```

Reply:
Nei det funker ikke. Klar til å merges denne. Det er en annen PR å fikse dette problemet